### DISCUSSION TITLE
[p.367] 백엔드 배포 질문입니다.

### DISCUSSION THREAD

**Discussion by KIMHAUN** [fsoftwareengineer/todo-application#102] [질문과 답변]
안녕하세요 책으로 웹 공부 정말 잘 하고 있는 학생입니다.
이제 AWS로 배포를 하려고 하는데 
![KakaoTalk_20220319_212148805](https://user-images.githubusercontent.com/33375516/159120958-ff10c7d1-fa03-45ed-988e-ffdc5ce5a810.png)
setenv와 다른 질문하신 분 처럼 eb deploy를 했을 때 
Successfully deployed new configuration to environment.라고 나왔습니다.(순서 반대로도 해봄)
그런데 일라스틱 빈스톡에서는 상태가 Warning이나 Degraded으로 나오길래 그냥 무시하고 하려 했는데 좀있으면 다시 Severe로 돌아가버립니다.
![KakaoTalk_20220319_211913170](https://user-images.githubusercontent.com/33375516/159121124-c28b787c-b96a-4820-9779-b05650e0c458.png)

심각의 원인은 
100.0 % of the requests are failing with HTTP 5xx.
Process default has been unhealthy for 16 hours (Target.ResponseCodeMismatch).
Following services are not running: application-web-1.

맨 처음 때는 세 번째 줄이 CPU를 100%사용하고 있다고 그랬는데 다시 해보니까 바뀌었습니다.
구글링을 해봤는데 디팬던시가 설정이 안됐다는 말도 있는데 runtimeOnly 'mysql:mysql-connector-java'도 넣어서 이건 아닌 것 같고
수차례 해봤는데 세 번째 줄만 바뀐 거 말고는 수확이 없습니다.
뭐를 어떻게 해야할까요? ㅠㅠ 
https://stackoverflow.com/questions/47195808/spring-boot-and-aws-elastic-beanstalk-following-services-are-not-running-appli

**Comment by rnjstmdals6** [comment]
저도 같은 문제로 진행이 안되고있습니다..

**Reply by rnjstmdals6** [reply]
플랫폼 상태 또한 Deprecated 로 뜨는데 찾아보니 리눅스2로 마이그레이션이 필요하다고 하는데 해결 방법을 알려주시면 대단히 감사하겠습니다 ㅠㅠ

**Comment by fsoftwareengineer** [comment]
프로파일설정이 PROD로 올바르게 올라가 있는지 확인 해 주시기 바랍니다.
https://github.com/fsoftwareengineer/todo-application/discussions/32#discussioncomment-1871283
